### ISSUE TITLE
`make install` doesn't install `tap`

### ISSUE THREAD

**Issue by david-bakin** [bitcoin-core/btcdeb#111]
I can get it installed with `make install-binPROGRAMS`.

(Sadly I don't know enough about automake tools to make a PR on this.)

**Comment by kallewoof** [comment]
Weird. On my end (Mac) it is installed fine:
```
$ make install
 ./build-aux/install-sh -c -d '/usr/local/bin'
  /bin/sh ./libtool   --mode=install /usr/bin/install -c btcdeb btcc test-btcdeb tap '/usr/local/bin'
libtool: install: /usr/bin/install -c btcdeb /usr/local/bin/btcdeb
libtool: install: /usr/bin/install -c btcc /usr/local/bin/btcc
libtool: install: /usr/bin/install -c test-btcdeb /usr/local/bin/test-btcdeb
libtool: install: /usr/bin/install -c tap /usr/local/bin/tap
make[2]: Nothing to be done for `install-data-am'.
```

What OS are you on?

**Comment by david-bakin** [comment]
linux, let me try again and record what happens.

**Comment by michaelfolkson** [comment]
@david-bakin: Did it work? Should this issue be closed?

**Comment by kallewoof** [comment]
I notice you said `make install-binPROGRAMS`. You usually just do `make install`. Try that and ping here if it doesn't work still.
